lib/service/config.ts

```diff
@@ -39,28 +49,72 @@ export default class Config {
 
   set(config) {
     // FIXME: Deep copy to avoid deleting itself.
-    config = JSON.parse(JSON.stringify(config))
+    const newConfig = { ...config }
     // FIXME: Treat as a mutable object always.
     //        We should always get config using `crowi.getConfig()` *just before* referencing config.
     for (const key of Object.keys(this.config)) {
       delete this.config[key]
     }
-    for (const key of Object.keys(config)) {
-      this.config[key] = config[key]
+    for (const key of Object.keys(newConfig)) {
+      this.config[key] = newConfig[key]
     }
```

Comment:
memo:
We can make it more simple like the following unless considering the direct assignment of the config object. But it is okay working on this ticket.
https://github.com/crowi/crowi/issues/685

```js
set(config) {
  this.config = config
}
```